src/pages/RecipeDetails.tsx

```diff
@@ -0,0 +1,121 @@
+import { useParams } from 'react-router';
+import { recipes } from '@/MockData/recipesMockData';
+import { NotFound } from './NotFound';
+
+export function RecipeDetails() {
+    const { id } = useParams();
+
+    const idNum = Number(id);
```

Comment:
Did you mean `isNum`?
```suggestion
    const isNum = Number(id);
```

Reply:
No. I am not just checking it if it is number or not. I want to convert `id`, which is in string to a number type.

Reply:
In that case, can you please provide more suitable name? This `idNum` seems bit misfit here.

Reply:
Provided more meaningful name.
https://github.com/dhruvdhaduk-simform/recipe-diary/pull/3/commits/aba6aaf18e80393b7d0d829e01f497ba92130fc8